WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -431,6 +440,47 @@ private extension StoreStatsAndTopPerformersViewController {
         stackView.addArrangedSubview(separator)
         return stackView
     }
+
+    func startCustomRangeTabCreation() {
+        guard let navigationController else { return }
+        customRangeCoordinator = CustomRangeTabCreationCoordinator(
+            navigationController: navigationController,
+            onDateRangeSelected: { [weak self] start, end in
+                self?.createCustomRangeTab(start, end)
+            }
+        )
+        customRangeCoordinator?.start()
+    }
+
+    func createCustomRangeTab(_ start: Date, _ end: Date) {
+        let currentDate = Date()
+
+        // TODO: 11935 Add the correct data fetching and displaying based on custom range.
+        let customRangeVC = StoreStatsAndTopPerformersPeriodViewController(siteID: siteID,
+                                                                           timeRange: .custom(from: start, to: end),
+                                                                           currentDate: currentDate,
+                                                                           canDisplayInAppFeedbackCard: true,
+                                                                           usageTracksEventEmitter: usageTracksEventEmitter)
+
+        periodVCs.append(customRangeVC)
+        timeRanges.append(.custom(from: start, to: end))
+
+        let customRangeTabbedItem = TabbedItem(title: Localization.customRangeTabTitle,
```

Comment:
We should declare the custom range into a local `let range` and use `range.tabTitle` here. 

`StatsTimeRangeV4` already provides the tab title for the custom range. `Localization.customRangeTabTitle` should be removed.

Reply:
TIL I missed that it already has `tabTitle`. Handled in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/bcd123be653bc29aa63bbe9507d339f8eea68a90